Bastien Olivier Dijkstra/BLOK2/HW1/BasicPhotoshopScript.jsx

```diff
@@ -0,0 +1,29 @@
+
+
+var orginalUnit = preferences.rulerUnits;
```

Comment:
Especially when working with vars, make sure to add some comments to make clear what you're doing in the script!

Reply:
Thank you, I was just playing around with photoshop script, didn't think comments where needed yet. Thanks for the feedback.